camera_name: azure_kinect
image_width: 1280
image_height: 720

camera_matrix:
  fx: 607.7908935546875
  fy: 607.75390625
  cx: 640.822509765625
  cy: 369.03350830078125